packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3459,8 +3459,10 @@ export default class Meeting extends StatelessWebexPlugin {
       this.owner =
         locusMeetingObject?.info.owner || meetingInfo?.owner || meetingInfo?.hostId || this.owner;
       this.permissionToken = meetingInfo?.permissionToken;
-      this.setPermissionTokenPayload(meetingInfo?.permissionToken);
-      this.setSelfUserPolicies();
+      if (this.permissionToken) {
+        this.setPermissionTokenPayload(meetingInfo?.permissionToken);
+        this.setSelfUserPolicies();
```

Comment:
any idea what does the setSelfUserPolicies do , any relation ship with permissionToken ?

Reply:
@arun3528 setPermissionTokenPayload decodes the jwt and then user policies are stored from the permission token payload.

Reply:
setSelfUserPolicies() uses optional chaining, so it should work even if permissionTokenPayload is undefined:
![image](https://github.com/webex/webex-js-sdk/assets/7473332/f9e3db1a-dcb3-4243-90d2-bf7ab105a664)